### PR TITLE
Clarify invites and expiry dates on users page

### PR DESCRIPTION
**Pull request by marians**
This PR separates user invites from user accounts to solve some usability problems.

## Preview

<details>
<summary>Before</summary>

image

</details>

<details>
<summary>After</summary>

image

</details>

### CODE REVIEW

File: src/components/Users/Users.js [hunk 1]
```diff
@@ -209,44 +201,44 @@ class Users extends React.Component {
...
+                <InvitesTable invitations={invitations} />
+                <Button onClick={this.inviteUser}>
+                  <i className='fa fa-add-circle' /> INVITE USER
+                </Button>
+              </>
+            </Section>
+            <Section title='User accounts'>
+              <UsersTable
+                users={users}
+                //invitationsAndUsers={invitationsAndUsers}
```
Comment (axbarsan) [inline src/components/Users/Users.js:224]: I believe this could be removed

File: src/components/Users/InvitesTable.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      dataField: 'expiry',
+      text: 'EXPIRES',
+      sort: true,
+      formatter: relativeDate,
+    },
+  ];
+};
+
+// eslint-disable-next-line react/no-multi-comp
+const InvitesTable = ({ invitations }) => {
```
Comment (axbarsan) [inline src/components/Users/InvitesTable.js:61]: Do you mind bringing a bit of TypeScript into this file? 😬

File: src/components/Users/Users.js (hunk 1 above)
Comment (marians) [reply src/components/Users/Users.js:224]: Indeed

File: src/components/Users/InvitesTable.js (hunk 2 above)
Comment (marians) [reply src/components/Users/InvitesTable.js:61]: In fact I do mind... but I guess I'll have to.

**Review by pipo02mix — APPROVED** [review]
LGTM from SE standpoint

**Comment by marians** [comment]
Status quo:

image

File: src/components/Users/InvitesTable.js (hunk 2 above)
Comment (marians) [reply src/components/Users/InvitesTable.js:61]: I tried, but failed with this problem:

image

**Review by oponder — APPROVED** [review]
LGTM

**Pull request merged**